### ISSUE TITLE
No info that card is needed not only to sign tx but also to log in and other typos on Complete screen 

### ISSUE THREAD

**Issue by annadanchenko** [status-im/status-react#7899]
# Problem

After setting up a card (creating PIN, etc) there is Complete screen with text "This card is now essential part your account security. Transactions can't be sent without it" 

There are 2.5 issues that need fixes:
1. Add "of", so it says "essential part of your account security"
2. Add extra sentence or rephrase existing one to say that you also need card to log in into the app (not only to sign transactions). I would rephrase and use same text as on one of Create PIN screens: "You'll need your card and PIN to log in and to confirm transactions."
3. I'm not sure if OKAY button is consistent with other similar buttons on the rest of the app screens, maybe change it to "Ok"?

![okay](https://user-images.githubusercontent.com/6584810/55498830-d2729400-5644-11e9-8b2a-48c06e73b5de.png)

**Comment by andmironov** [comment]
Addressed in https://github.com/status-im/status-react/issues/9264
